sql/dql/basic-queries/order-by-clause.md

```diff
@@ -41,117 +41,104 @@ aspects:
 ```sql
-SELECT COUNT(name), language_id
-FROM move_name
-GROUP BY language_id;
+SELECT last_name, first_name
+FROM employees
+ORDER BY last_name;
 ```
 
-*Result:*
-```
-count | language_id
-======+============
-  639 |           6
-  639 |           5
-  639 |           8
-  639 |           1
-  639 |           9
-  617 |           3
-  639 |           7
-(7 rows)
-```
+*Resulting set:*
+
+| last_name | first_name |
+| --------- | ---------- |
+| Beeho     | Donnamarie |
+| Isworth   | Tony       |
+| Kippen    | Immanuel   |
+| Pantone   | Karlene    |
+| Pantone   | Regina     |
+| Warrier   | Esta       |
+
+Note that the ascending order is set as default. If we want to display the result in a descending order we need to put `DESC` after specifying the columns.
+
+There might be some cases where you want your result set to be ordered by multiple columns, each being sorted differently (ascending or descending). In SQL, this is done by specifying the name of the columns you want ordered, followed by their order. On top of that, each set of column-order should be separated by a comma. When you're sorting using only one column, the result set might contain duplicate values. In this case, you can add another column to the sorting strategy such that the duplicate values would be sorted by the second column.
+
+On the same `employees` table, to return the `last_name` and `first_name`, sorted in ascending order by the `last_name` and in descending order by the `first_name`, the command would be:
 
-Now using `ORDER BY` clause:
 ```sql
-SELECT COUNT(name), language_id
-FROM move_name
-GROUP BY language_id
-ORDER BY language_id;
+SELECT last_name, first_name
+FROM employees
+ORDER BY last_name ASC, first_name DESC;
 ```
 
-*Result:*
-```
-count | language_id
-======+============
-  639 |           1
-  617 |           3
-  639 |           5
-  639 |           6
-  639 |           7
-  639 |           8
-  639 |           9
-(7 rows)
-```
+*Resulting set*:
 
-Note that the ascending order is set as default. If we want to display the result in a descending order we need to put `DESC` after specifying the columns.
+| last_name | first_name |
+| --------- | ---------- |
+| Beeho     | Donnamarie |
+| Isworth   | Tony       |
+| Kippen    | Immanuel   |
+| Pantone   | Regina     |
+| Pantone   | Karlene    |
+| Warrier   | Esta       |
+
+First, the `last_name` column is sorted in ascending order, and for the rows where the `last_name` have equal values ('Pantone Karlene', 'Pantone Regina') the `first_name` column is sorted in descending order.
 
 ---
 ## Practice
 
-The `nature_name` table has data about the nature name in multiple languages. We have the fields:
-`name`, for nature name and `language_id` to reflect the given language.
-Count how many names we have for each language and order the result in descending order:
+The `nature_name` table contains data about the nature name in multiple languages, and it has the following columns:
```

Comment:
"nature name" -> "different equivalents of the english 'nature' word" ?

Reply:
changed to

The `students` table contains data about each student at an international school. Two of the columns in the table are represented by:
* `name`, for the student's name
* `language_id` to reflect the native language